ruby_on_rails/advanced_forms_and_activerecord/forms_advanced.md

```diff
@@ -32,7 +32,11 @@ Let's say you want to build a New Post form for your blog but you want to be abl
 
 The bare HTML way is to build a bunch of `<option>` tags (inside your `<select>` tag).  You could easily create these in your ERB code by just iterating over some collection, for instance if you'd like to select a post to view from a list of them.
 
+<<<<<<< HEAD
+```html
+=======
 ```erb
+>>>>>>> main
```

Comment:
Lines 35-37 need to be deleted. Line 38 also needs to be deleted. This is all just to fully resolve the merge conflict.
```
<<<<<< HEAD
This section shows things that are in your local branch that
are different from what's on `main`. This is because your
branch had become out of date. We want to get rid of 
all of this and accept the stuff that's in `main`
======

>>>>>>> main
This stuff is content that's a part of `main`.
You want to accept these changes, but you still need
to get rid of the annotation lines `git` has placed
surrounding this content. Looks like you already got
rid of the `=======` bit, but `>>>>>> main` is still
here.
=======
```
Once you do this, this file should match `main` and will stop showing in the diff for this PR. Which is good because this file has nothing to do with the PR you're making here.